rosdep/python.yaml

```diff
@@ -6990,6 +6990,16 @@ python3-geographiclib:
     '*': [python3-GeographicLib]
     '7': null
   ubuntu: [python3-geographiclib]
+python3-geojson-pip:
+  debian:
+    pip:
+      packages: [geojson]
+  fedora:
+    pip:
+      packages: [geojson]
+  ubuntu:
+    pip:
+      packages: [geojson]
```

Comment:
So it looks like geojson is actually available as system packages for all of these distributions:

* Debian: https://packages.debian.org/search?keywords=python3-geojson&searchon=names&suite=all&section=all
* Ubuntu: https://packages.ubuntu.com/search?keywords=python3-geojson&searchon=names&suite=all&section=all
* Fedora: https://packages.fedoraproject.org/pkgs/python-geojson/python3-geojson/

Please update the key to be a regular one (`python3-geojson`), and have it reference the system packages.

Reply:
Okay, I've updated the key to reference the system packages. I also found this package on opensuse and arch. Can you take another look? Thanks